lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -339,7 +339,8 @@ export abstract class ClientApplication {
                 throw BrowserAuthError.createNoCachedAuthorityError();
             }
 
-            const authClient = await this.createAuthCodeClient(serverTelemetryManager, currentAuthority);
+            const authClient = await this.createAuthCodeClient(serverTelemetryManager, currentAuthority, cachedRequest.correlationId);
+            authClient.logger.verbose("Auth code client created", "", name, version);
```

Comment:
It's not entirely clear to me why we need to use the authClient logger here instead of msal-browsers logger? Wouldn't it be cleaner to just call `authClient.logger.verbose("Auth code client created", cachedRequest.correlationId);` and not have to add name and version to the log messages themselves?